manager/migrations/0003_auto_20180718_1444.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-07-18 14:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+from django.core.management import call_command
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('manager', '0002_recipientgroup_archived'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SubscriptionCategory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text=b'The name of the subscription category. Will be viewed by users on frontend.', max_length=100, unique=True)),
```

Comment:
Small grammar-y suggestion: update 2nd sentence of 'name' help text to "Will be viewed by users on _the_ frontend" or "Will be viewable by front-end users."